server/util/twilio.js

```diff
@@ -1,10 +1,19 @@
 import Twilio from 'twilio';
+import dotenv from 'dotenv';
 
+dotenv.load();
```

Comment:
Dotenv is *OUT* make sure you're setting your env variables in your bash profile.